src/Symfony/Component/Messenger/Tests/Stamp/ValidationStampTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Tests\Stamp;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Messenger\Stamp\ValidationStamp;
+use Symfony\Component\Validator\Constraints\GroupSequence;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class ValidationStampTest extends TestCase
+{
+    public function testStamp()
+    {
+        $stamp = new ValidationStamp($groups = array('Default', 'Extra'));
+        $this->assertSame($groups, $stamp->getGroups());
+
+        $stamp = new ValidationStamp($groups = new GroupSequence(array('Default', 'Then')));
+        $this->assertSame($groups, $stamp->getGroups());
+    }
+
+    public function testSerializable()
```

Comment:
A changelog entry is missing to explain this, but since #28270, we don't expect anymore the stamps to be serializable-safe using PHP, as we'll always use the Serializer component.
Same for other stamps tests.

Reply:
I think that's still fine telling and testing as a general rule.